src/platforms/stm32/src/lib/gpiodriver.c

```diff
@@ -96,13 +108,20 @@ static void consume_gpio_mailbox(Context *ctx)
         }
 
     } else {
-        TRACE("gpio: unrecognized command\n");
+        TRACE("gpio: unrecognized command :: %s\n", cmd);
```

Comment:
Good eye! Didn't spot that mistake lurking there... This whole driver will need a near complete rewrite. Currently all of the GPIO atoms are in the platform_defaultatoms.h and should be moved  into the driver itself. Many commands and all of the nif functions need to be added. I plan on using the esp32 gpio_driver.c as a template for the rewrite, and will try to keep the code as consistent across platforms as possible.

Reply:
Actually in this case I believe "cmd" will not be an integer, since the command will not likely match an atom, it is possible it will match an atom, but not guaranteed, or even likely in this situation...

Reply:
reverted this change.